src/pages/playground/index.tsx

```diff
@@ -230,89 +244,103 @@ function Playground() {
         cellStyle: {
-          backgroundColor: usedIdentificationEvent?.products?.ipBlocklist?.data?.result === true ? RED : GREEN,
+          backgroundColor:
+            usedIdentificationEvent?.products?.ipBlocklist?.data?.result ||
+            usedIdentificationEvent?.products?.proxy?.data?.result ||
+            usedIdentificationEvent?.products?.tor?.data?.result
+              ? RED
+              : GREEN,
         },
       },
     ],
     [
       {
-        content: ['Proxy', <Info key="info">The request IP address is used by a public proxy provider.</Info>],
-      },
-      {
-        content:
-          usedIdentificationEvent?.products?.proxy?.data?.result === true ? 'You are using a proxy 🔄' : 'Not detected',
-        cellStyle: { backgroundColor: usedIdentificationEvent?.products?.proxy?.data?.result === true ? RED : GREEN },
+        content: [
+          <DocsLink href="https://dev.fingerprint.com/docs/smart-signals-overview#raw-device-attributes">
+            Raw device attributes
+          </DocsLink>,
+        ],
       },
+      { content: 'Applicable only to browsers. See the JSON below.', cellStyle: { backgroundColor: GRAY } },
     ],
     [
       {
-        content: ['Tor Network', <Info key="info">The request IP address is a known Tor network exit node.</Info>],
-      },
-      {
-        content:
-          usedIdentificationEvent?.products?.tor?.data?.result === true ? 'You are using Tor 🧅' : 'Not detected',
-        cellStyle: { backgroundColor: usedIdentificationEvent?.products?.tor?.data?.result === true ? RED : GREEN },
+        content: [
+          <DocsLink href="https://dev.fingerprint.com/docs/smart-signals-overview#frida-detection">
+            App is instrumented by Frida
+          </DocsLink>,
+        ],
       },
+      { content: 'Applicable only for iOS and Android devices', cellStyle: { backgroundColor: GRAY } },
     ],
     [
       {
-        content: ['Android Emulator', <Info key="info">Android specific emulator detection.</Info>],
+        content: [
+          <DocsLink href="https://dev.fingerprint.com/docs/smart-signals-overview#factory-reset-detection">
+            Factory Reset Timestamp
+          </DocsLink>,
+        ],
       },
-      { content: 'Not applicable to browsers', cellStyle: { backgroundColor: GRAY } },
+      { content: 'Applicable only for iOS and Android devices', cellStyle: { backgroundColor: GRAY } },
     ],
     [
       {
         content: [
-          'Android Tampering',
-          <Info key="info">Android specific root management apps detection, for example, Magisk.</Info>,
+          <DocsLink href="https://dev.fingerprint.com/docs/smart-signals-overview#geolocation-spoofing-detection">
+            Location spoofing
+          </DocsLink>,
         ],
       },
-      { content: 'Not applicable to browsers', cellStyle: { backgroundColor: GRAY } },
+      { content: 'Applicable only for iOS and Android devices', cellStyle: { backgroundColor: GRAY } },
```

Comment:
I'm worried, maybe we should describe such repeating statuses in one place and reuse them?

Reply:
Good point, done :)